### ISSUE TITLE
>=0.10 engines claim is (probably) no longer accurate

### ISSUE THREAD

**Issue by cburroughs** [TritonDataCenter/node-manta#365]
Currently package.json states:

```
    "engines": {
        "node": ">=0.10"
}
```

When I try to `npm install` with v0.10.48 I get

```
npm WARN engine mime@1.6.0: wanted: {"node":">=4"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine tap@12.6.1: wanted: {"node":">=4"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine coveralls@3.0.3: wanted: {"node":">=4.0.0"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine nyc@13.3.0: wanted: {"node":">=6"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine esm@3.2.22: wanted: {"node":">=6"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine ts-node@8.0.3: wanted: {"node":">=4.2.0"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine typescript@3.4.1: wanted: {"node":">=4.2.0"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine growl@1.10.5: wanted: {"node":">=4.x"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine request@2.88.0: wanted: {"node":">= 4"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine yn@3.0.0: wanted: {"node":">=6"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine esprima@4.0.1: wanted: {"node":">=4"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine form-data@2.3.3: wanted: {"node":">= 0.12"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine har-validator@5.1.3: wanted: {"node":">=6"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine har-schema@2.0.0: wanted: {"node":">=4"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine istanbul-lib-instrument@3.1.0: wanted: {"node":">=6"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine punycode@2.1.1: wanted: {"node":">=6"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine @babel/parser@7.4.3: wanted: {"node":">=6.0.0"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine jsesc@2.5.2: wanted: {"node":">=4"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine globals@11.11.0: wanted: {"node":">=4"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine to-fast-properties@2.0.0: wanted: {"node":">=4"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine chalk@2.4.2: wanted: {"node":">=4"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine chalk@2.4.2: wanted: {"node":">=4"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine supports-color@5.5.0: wanted: {"node":">=4"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine ansi-styles@3.2.1: wanted: {"node":">=4"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine supports-color@5.5.0: wanted: {"node":">=4"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine ansi-styles@3.2.1: wanted: {"node":">=4"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine has-flag@3.0.0: wanted: {"node":">=4"} (current: {"node":"0.10.48","npm":"2.15.1"})
npm WARN engine has-flag@3.0.0: wanted: {"node":">=4"} (current: {"node":"0.10.48","npm":"2.15.1"})
```

It's *possible* that node-manta still works in some cases, but given the test tools are among the above errors, it is hard to be confident in that.

I'm unsure what a better value would be, or how long this library needs to continue to support EOL nodejs versions.

**Comment by davepacheco** [comment]
FWIW, I believe we still use this package with Node v0.10 in a number of places.  Sadly, much of Manta is still on 0.10.

**Comment by bahamat** [comment]
Fixed in #396
